docs/install/installation.rst

```diff
@@ -1,11 +1,13 @@
 .. meta::
    :description: A library that can be loaded by ROCr to print the AMDGPU wavefront states
-   :keywords: ROCdebug-agent, ROCm, library, tool, rocr
+   :keywords: ROCdebug-agent installation, ROCR Debug Agent installation, install ROCdebug-agent, install ROCR Debug Agent,
+    build ROCdebug-agent, build ROCR Debug Agent
+
 
 .. _installation:
 
 ==================================
-Installation
+ROCR debug agent installation
 ==================================
```

Comment:
The length of  the `===[…]===` lines seems off.

Reply:
@lancesix I didn't get this one. The length of ==== line seems uniform to me.

Reply:
Couldn't it be:

```suggestion
=============================
ROCR debug agent installation
=============================
```
i.e. 4 less "=", so it matches the width of the title?

note that this is not a hard requirement, this is just pure cosmetics.  The rendered output should still be the same regardless